Updated environment and Pandana 0.4 compatibility

smmaurer opened the pull request:
This PR has several small fixes:

1. ~~Adds `cast = True` in `update_col_from_series()`, in the `household_relocation` step. I was getting an int32/int64 error there.~~ (Already fixed in another PR.)

2. ~~Replaces `settings` injectable with newer `policy` injectable in one spot in `summaries.py` -- I was getting an error that `settings` doesn't exist, because it's no longer passed into that model step. (Everything else in that model step uses `policy`; maybe this line was broken when I was resolving conflicts during the merges I made a couple of months ago?) But @theocharides please confirm that this looks right!~~ (Already fixed in another PR.)

3. Adjusts some Pandana aggregation settings -- it turns out that 'average' was an undocumented alias for 'ave', and 'average' was accidentally removed in Pandana v0.4. Using 'ave' will work with all Pandana versions. See https://github.com/UDST/pandana/issues/133.

4. Environments: Updates `requirements-2019.txt` to improve support for Python 2.7. 

5. Adds a Conda environment file: `baus-env-2020.yml`. This should be the default now.  

6. Updates main README.md with new installation instructions.

theocharides commented:
Hey @smmaurer- looks like you already caught that #1 was fixed in master, as was #2, prior to this PR.

smmaurer commented:
Hi @theocharides, this is ready to merge! I've added installation instructions, and updated the title and original writeup.

My only uncertainty is whether we should still include `python baus.py --mode preprocessing` in the installation instructions -- see Slack. I've left it out for now, but we can add it back if needed.

smmaurer commented:
Ok, I've added back the preprocessing to the instructions! There's a bug preventing that from running at the moment, which I'll fix in another PR. But no need to wait on that for merging this one, @theocharides.

theocharides commented:
Confirmed that the Conda environment works in Linux (in addition to @smmaurer's Mac testing). Will likely remove the need for a pip install option going forward.
